addons/api/mirage/factories/target.js

```diff
@@ -22,25 +22,23 @@ export default factory.extend({
       'add-credential-sources',
       'remove-credential-sources',
     ],
-
-  id: () => generateId('t_'),
+  id() {
+    return this.type === TYPE_TARGET_TCP
+      ? generateId('ttcp_')
+      : generateId('tssh_');
+  },
 
   /**
    * -1 means "unlimited" and we want to generate these on occasion.
    */
   session_connection_limit: () =>
     faker.helpers.arrayElement([-1, faker.datatype.number()]),
   worker_filter: (i) => (i % 2 === 0 ? faker.random.words() : null),
+  egress_worker_filter: (i) => (i % 2 !== 0 ? faker.random.words() : null),
+  ingress_worker_filter: (i) => (i % 2 !== 0 ? faker.random.words() : null),
```

Comment:
In the API, `worker_filter` is mutually exclusive with the new fields.  If it is set, the other two _may not_ be set, and vice versa.  And this mutual exclusivity is one of the assumptions used to drive the new migration UI.  I recommend reflecting this constraint in the mock data.

Reply:
<img width="431" alt="Screen Shot 2023-01-09 at 10 22 16 AM" src="https://user-images.githubusercontent.com/29386339/211356652-9182ddf2-e8b0-4b36-a075-dc7aa6d4d806.png">
This is how the data is being generated with these mirage changes. This should satisfy the constraint between `worker_filter` and the new properties.